python/plugins/processing/algs/qgis/SnapGeometries.py

```diff
@@ -119,6 +124,10 @@ def processAlgorithm(self, parameters, context, feedback):
                     sink.addFeature(f)
                 processed += 1
                 feedback.setProgress(processed * total)
+        elif mode == 7:
+            # input layer == ref layer
+            modified_count = QgsGeometrySnapperSingleSource.run(source, sink, tolerance, feedback)
+            feedback.pushInfo(self.tr('Snapped {} geometries.').format(modified_count))
```

Comment:
Ideally this would be a numeric output from the alg, but the other modes prevent that.






^^ thinking aloud only